style/layout.css

```diff
@@ -0,0 +1,88 @@
+header #a {

```

Comment:
since you only have one `#a`, you don't need to specify the one in `header`

Reply:
fixed!